src/state/pools/fetchPools.ts

```diff
@@ -102,11 +103,16 @@ export const fetchPoolsStakingLimits = async (
     .filter((p) => !poolsWithStakingLimit.includes(p.sousId))
 
   // Get the staking limit for each valid pool
-  // Note: We cannot batch the calls via multicall because V1 pools do not have "poolLimitPerUser" and will throw an error
-  const stakingLimitPromises = validPools.map((validPool) => fetchPoolStakingLimit(validPool.sousId))
-  const stakingLimits = await Promise.all(stakingLimitPromises)
-
-  return stakingLimits.reduce((accum, stakingLimit, index) => {
+  const poolStakingCalls = validPools.map((validPool) => {
+    const config = poolsConfig.find((pool) => pool.sousId === validPool.sousId)
```

Comment:
`validPools` is filtered from `poolsConfig`. Why do we need to `poolsConfig.find` again?

Reply:
Done